libraries/core_libs/network/src/tarcap/packets_handlers/get_pbft_sync_packet_handler.cpp

```diff
@@ -32,38 +32,58 @@ void GetPbftSyncPacketHandler::process(const PacketData &packet_data,
   // Here need PBFT chain size, not synced period since synced blocks has not verified yet.
   const size_t my_chain_size = pbft_chain_->getPbftChainSize();
   size_t blocks_to_transfer = 0;
+  auto will_be_synced = false;
```

Comment:
this flag seems like a redundant info to what we are already sending -> last_block fklag

Reply:
They are different.
last_block flag is the last block in each syncing round
will_be_synced is the flag to tell peer its PBFT chain has synced with mine
For example:
Node1 has PBFT chain size 7, node2 has pbft chain size 0, and network_sync_level_size_ is 5
Round 1:
node1 send 5 PBFT blocks [1, 2, 3, 4, 5] to node2
when send block5, the last_block is true, but will_be_synced is false
Round 2:
node1 send 2 PBFT blocks [6, 7] to node2
when send block7, the last_block is true and will_be_synced is also true

Reply:
hmm ok I understand now. Maybe better name instead of `will_be_synced` is `pbft_synced`. Once you receive packet with this flag set to true, sender tells you that with this packet you are pbft_synced with him...

Reply:
Rename will_be_synced to pbft_chain_synced